webapp/channels/src/components/admin_console/blockable_link/blockable_link.tsx

```diff
@@ -24,29 +24,29 @@ type Props = {
     className?: string;
     onClick?: (e: React.MouseEvent) => void;
 };
-export default class BlockableLink extends React.PureComponent<Props> {
-    private handleClick = (e: React.MouseEvent) => {
-        if (this.props.onClick) {
-            this.props.onClick(e);
+
+const BlockableLink = (props: Props) => {
+    const {blocked, actions, ...restProps} = props;
+    const linkProps = {...restProps};
```

Comment:
you can this in the parameters itself of the function

Reply:
Done.